LICENSE.txt

```diff
@@ -1,21 +1,24 @@
-The MIT License (MIT)
+# Copyright (c) 2018, TU Wien, Department of Geodesy and Geoinformation
```

Comment:
I would have expected that the copywrite is the same as that in the esa cci sm package: https://github.com/TUW-GEO/esa_cci_sm/blob/master/esa_cci_sm/interface.py